src/core/Internal.ml

```diff
@@ -1212,13 +1212,13 @@ module Make (Plugin : PLUGIN) = struct
       | [_] ->
         if history = [] then
           (* no simplification has been done, so [cl] is actually a clause with only
-               [a], so it is a valid reason for propagating [a]. *)
+                 [a], so it is a valid reason for propagating [a]. *)
```

Comment:
Mutli-line comments should be aligned with the first letter after the '*' I think